tests/unit/src/test/scala/tests/cli/ScalacOptionsSuite.scala

```diff
@@ -98,4 +98,26 @@ class ScalacOptionsSuite extends BaseCliSuite {
       assert(!out.contains("discarded non-Unit value"))
     }
   )
+
+  checkCli(
+    "no-imports",
+    """
+      |/index.md
+      |```scala mdoc
+      |scala.List(1)
+      |```
+      |""".stripMargin,
+    """|/index.md
+       |```scala
+       |scala.List(1)
+       |// res0: scala.collection.immutable.List[scala.Int] = List(1)
```

Comment:
Thanks @davidhoyt for tracking down the issue! Note that the printed output takes into account the scope so `scala.Int` is printed out instead of `Int`